datastore/src/main/java/org/jboss/pnc/datastore/DefaultDatastore.java

```diff
@@ -194,17 +216,27 @@ private Artifact saveHttpArtifacts(Artifact artifact) {
         Artifact artifactFromDb;
         artifactFromDb = artifactRepository
                 .queryByPredicates(withOriginUrl(artifact.getOriginUrl()));
+
         if (artifactFromDb == null) {
+            logger.trace("Artifact is not in DB. Saving artifact {}.", artifact);
+
             // remove checksum of http downloads because thay are mutable
             // persistent http downloads should be stored per build and not shared
             artifact.setSha1("");
             artifact.setMd5("");
             artifact.setSha256("");
+
             //Relation owner (BuildRecord) must be saved first, the relation is saved when the BR is saved
             artifact.setBuildRecords(Collections.emptySet());
             artifact.setDependantBuildRecords(Collections.emptySet());
             artifactFromDb = artifactRepository.save(artifact);
+
+            logger.trace("Saved new artifact {}.", artifactFromDb);
         }
```

Comment:
!

Reply:
What's the problem here? If you mean the empty line, it's intentional to make it easily-readable.